### PR TITLE
update supported Ubuntu version

### PR DESCRIPTION
**Pull request by bradgrantham-lunarg**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -236,6 +236,8 @@ Building on Linux requires the installation of the following packages:
 
 #### Ubuntu
 
+GFXReconstruct is built and tested for SDK development using Ubuntu 20.04.  Older Ubuntu releases are unsupported but may work.
```
Comment (andrew-lunarg) [inline BUILD.md:239]: " but may work" implies "may not work", which makes these words carry zero actionable information, and suggests deletion of them.

**Review by andrew-lunarg — CHANGES_REQUESTED** [review]
We aren't giving support so why are we giving hope?

File: BUILD.md (hunk 1 above)
Comment (bradgrantham-lunarg) [reply BUILD.md]: Yes!

File: BUILD.md [hunk 2]
```diff
@@ -236,7 +236,7 @@ Building on Linux requires the installation of the following packages:
 
 #### Ubuntu
 
-GFXReconstruct is built and tested for SDK development using Ubuntu 20.04.  Older Ubuntu releases are unsupported but may work.
+GFXReconstruct is built and tested for SDK development using Ubuntu 20.04.  Older Ubuntu releases are unsupported.
```
Comment (andrew-lunarg) [inline BUILD.md:239]: Beautifully succinct!

**Review by andrew-lunarg — APPROVED** [review]

**Pull request merged**